language/move-lang/src/parser/byte_string.rs

```diff
@@ -0,0 +1,355 @@
+                    self.escape_sequence = None;
+                    match ch {
+                        'n' => self.push('\n'),
+                        'r' => self.push('\r'),
+                        't' => self.push('\t'),
+                        '\\' => self.push('\\'),
+                        '0' => self.push('\0'),
+                        '"' => self.push('"'),
+                        'x' => self.escape_sequence = Some(Sequence::Hex(String::new())),
+                        'u' => self.escape_sequence = Some(Sequence::Unicode(String::new())),
+                        '\n' => self.escape_sequence = Some(Sequence::Linebreak),
+                        _ => {
+                            return self.err(
+                                idx - 1,
+                                idx,
+                                format!("invalid escape sequence: '\\{}'", ch),
+                            );
+                        }
+                    }
+                }
+                Sequence::Hex(hex) => {
+                    hex.push(ch);
+                    if hex.len() == 2 {
+                        if let Ok(buffer) = hex::decode(&hex) {
+                            self.buffer.extend(buffer);
+                        } else {
+                            let err_text = format!(
+                                "invalid character in numeric character escape: '\\x{}'",
+                                hex
+                            );
+                            return self.err(idx - 3, idx, err_text);
+                        }
+                        self.escape_sequence = None;
+                    }
+                }
+                Sequence::Unicode(unicode) => {
+                    unicode.push(ch);
+
+                    let len = unicode.len();
+                    if len == 1 && !unicode.starts_with('{') {
+                        let err_test = format!("invalid unicode escape: '\\u{}'", unicode);
+                        return self.err(idx - 2, idx, err_test);
+                    }
+
+                    if len > 8 {
+                        return self.err(
+                            idx - len - 1,
+                            idx,
+                            "unicode escape must be at most 10FFFF".to_string(),
+                        );
+                    }
+
+                    if ch == '}' {
+                        if let Ok(buffer) = hex::decode(format!("{:0>8}", &unicode[1..len - 1])) {
+                            if let Ok(ch) = char::try_from(BigEndian::read_u32(&buffer)) {
+                                self.push(ch);
+                            } else {
+                                let err_test = format!("invalid unicode escape: '\\u{}'", unicode);
+                                return self.err(idx - len - 1, idx, err_test);
+                            }
+                        } else {
+                            let err_test = format!("invalid unicode escape: '\\u{}'", unicode);
```

Comment:
Capitalize "Invalid" (and also "Unicode" since it is a proper noun)

Reply:
Done.